recipes-pv/pantavisor/files/fw_env.config

```diff
@@ -0,0 +1,11 @@
+# Configuration file for fw_(printenv/setenv) utility.
+# Up to two entries are valid, in this case the redundant
+# environment sector is assumed present.
+# Notice, that the "Number of sectors" is not required on NOR and SPI-dataflash.
+# Futhermore, if the Flash sector size is omitted, this value is assumed to
+# be the same as the Environment size, which is valid for NOR and SPI-dataflash
+# Device offset must be prefixed with 0x to be parsed as a hexadecimal value.
+
+# Device name           Device offset   Env. size       Flash sector size       Number of sectors
+/dev/mtd1               0               0x20000         0x20000                 4
```

Comment:
This config is specific for devices with nand  flash. How about others which used emmc/SD?